std_msgs/Header header
int32 uav_id_from
int32 fresh
float64 detect_R
float32 resolution
OccupyData[] map